spec/services/bgs/dependent_service_spec.rb

```diff
@@ -4,6 +4,7 @@
 
 RSpec.describe BGS::DependentService do
   let(:user) { FactoryBot.create(:evss_user, :loa3, birth_date:) }
+  let(:user2) { FactoryBot.create(:evss_user_no_participant, :loa3, birth_date:) }
```

Comment:
If you do this then you don't need to create another factory: 
`let(:user2) { FactoryBot.create(:evss_user, :loa3, participant_id: nil, birth_date:) }`